src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/CrazyPriceHost.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Exadel.CrazyPrice.Common.Configurations;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.IO;
+
+namespace Exadel.CrazyPrice.Common
+{
+    public static class CrazyPriceHost
+    {
+        public static void CheckLoggerAndStartHost(IHostBuilder hostBuilder,
+            string initMessage, string errorInitMessage)
```

Comment:
Why you need init<essage/initError messages

It makes sense to run Logger only here - if something get wrong exception will raised and app will stop. 
Run in separate place. Rename method as well

Reply:
Removed.